### PR TITLE
11216 cards refresh

### PR DESCRIPTION
**Pull request by petrican**
https://github.com/metasfresh/metasfresh/issues/11216

https://www.loom.com/share/65e5f1de9980491fa056c3c7d07aeb0f

### CODE REVIEW

File: frontend/src/containers/Board.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ class Board extends Component {
       addCard(board.boardId, targetLaneId, card.id, card.index);
 
       if (this.sideNav) {
-        this.sideNav.removeCard(card.id);
+        this.sideNav.instanceRef.refreshView(board.boardId);
```
Comment (teosarca) [inline frontend/src/containers/Board.js:125]: IMHO you shall wait for addCard operation to finish to increase the chance of refreshing AFTER it was changed in backend.

But now that I am looking at the code, I believe the quickest workaround is to make the removeCard work, i.e.
```javascript
this.sideNav.removeCard(card.id);
```

**Review by teosarca — CHANGES_REQUESTED** [review]
check my comments

File: frontend/src/containers/Board.js (hunk 1 above)
Comment (petrican) [reply frontend/src/containers/Board.js:125]: Tried that and it does not work as it should. More debugging and more time needed to make it work with removeCard.

Comment (petrican) [reply frontend/src/containers/Board.js]: moved this to be triggered with the ws event

**Pull request merged**